[2025-07-28T12:44:36.932+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_pipeline.dbt_deps manual__2025-07-28T12:44:30.180992+00:00 [queued]>
[2025-07-28T12:44:36.935+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_pipeline.dbt_deps manual__2025-07-28T12:44:30.180992+00:00 [queued]>
[2025-07-28T12:44:36.935+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-07-28T12:44:36.939+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): dbt_deps> on 2025-07-28 12:44:30.180992+00:00
[2025-07-28T12:44:36.942+0000] {standard_task_runner.py:60} INFO - Started process 107 to run task
[2025-07-28T12:44:36.943+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dbt_pipeline', 'dbt_deps', 'manual__2025-07-28T12:44:30.180992+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/dbt_pipeline.py', '--cfg-path', '/tmp/tmpi0c9jq7t']
[2025-07-28T12:44:36.945+0000] {standard_task_runner.py:88} INFO - Job 39: Subtask dbt_deps
[2025-07-28T12:44:36.950+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-07-28T12:44:36.971+0000] {task_command.py:423} INFO - Running <TaskInstance: dbt_pipeline.dbt_deps manual__2025-07-28T12:44:30.180992+00:00 [running]> on host 6ed405011aee
[2025-07-28T12:44:37.000+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gotphoto' AIRFLOW_CTX_DAG_ID='dbt_pipeline' AIRFLOW_CTX_TASK_ID='dbt_deps' AIRFLOW_CTX_EXECUTION_DATE='2025-07-28T12:44:30.180992+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-28T12:44:30.180992+00:00'
[2025-07-28T12:44:37.001+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-07-28T12:44:37.001+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/dbt_project && dbt deps --project-dir . --profiles-dir profiles']
[2025-07-28T12:44:37.004+0000] {subprocess.py:86} INFO - Output:
[2025-07-28T12:44:37.548+0000] {subprocess.py:93} INFO - [0m12:44:37  Running with dbt=1.8.7
[2025-07-28T12:44:38.241+0000] {subprocess.py:93} INFO - [0m12:44:38  Installing dbt-labs/dbt_utils
[2025-07-28T12:44:38.756+0000] {subprocess.py:93} INFO - [0m12:44:38  Installed from version 1.1.1
[2025-07-28T12:44:38.756+0000] {subprocess.py:93} INFO - [0m12:44:38  Updated version available: 1.3.0
[2025-07-28T12:44:38.757+0000] {subprocess.py:93} INFO - [0m12:44:38
[2025-07-28T12:44:38.757+0000] {subprocess.py:93} INFO - [0m12:44:38  Updates available for packages: ['dbt-labs/dbt_utils']
[2025-07-28T12:44:38.758+0000] {subprocess.py:93} INFO - Update your versions in packages.yml, then run dbt deps
[2025-07-28T12:44:39.362+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-07-28T12:44:39.375+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dbt_pipeline, task_id=dbt_deps, execution_date=20250728T124430, start_date=20250728T124436, end_date=20250728T124439
[2025-07-28T12:44:39.418+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-07-28T12:44:39.433+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
